docs/model_library/strategic_water_management/index.rst

```diff
@@ -1250,7 +1250,7 @@ For, this if desalination technology is not selected then amount of desalinated
 .. math::
 
     \textcolor{red}{F_{r,t}^{DesalinatedWater}}
-        \leq \textcolor{green}{M^{Flow}} \cdot (1 -
+        \leq \textcolor{green}{M^{Flow}} \cdot (
```

Comment:
Please remove the first brackets around the summation term. (Minor comment) edit the sentence above the constraint "For, this ..." to "For this, ..."